src/components/_expansion-card.scss

```
@@ -86,7 +172,7 @@ category: Components
 }
 
 .ncgr-expansion-card__contents {
```

[IMO]これもともとこうなってたからあれなんだけど、上がheaderときてるから、bodyとかの方がいいかもしれない

```suggestion
.ncgr-expansion-card__body {
```